manpages/lumina-fileinfo.8

```diff
@@ -0,0 +1,27 @@
+.Dd October 24, 2017
+.Dt LUMINA-FILEINFO 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-fileinfo
+.Nd is used to view the properties of specific files. It also includes a simple XDG application registration creator.
+
+.Sh SYNOPSIS
+.Nm
+.Op file
+
+.Sh DESCRIPTION
+.Nm
+displays a graphical utility for viewing/changing the properties of a file. It yields
+detailed information about the specifed file such as type, mimetype,
+file size, owner, group, permissions, created, and last modified.
```

Comment:
-> file details including type, mimetype, file size, owner, group, permissions, date created, and date of last modification.